### PR TITLE
fix/iSM996

### PR DESCRIPTION
**Pull request by IVANDOMENZAIN**
An empty `prot_abundance.txt` file has been added to `eciSM996/scripts` as discussed in [PR#124](https://github.com/SysBioChalmers/GECKO/pull/124#issuecomment-749556348) in GECKO. This change enables generation of a functional ecModel for iSM996 and all outputs comply with the expected format and content.

### CODE REVIEW

**Comment by mihai-sysbio** [comment]
I see the printouts in `manualModifications.m` are not present for other models (eg [here](https://github.com/MetabolicAtlas/ecModels/blob/feat/updated-models/ecHumanGEM/scripts/manualModifications.m)). Would it make sense to remove them from there so they don't interrupt the Matlab log?

**Review by mihai-sysbio — APPROVED** [review]

**Comment by IVANDOMENZAIN** [comment]
> I see the printouts in `manualModifications.m` are not present for other models (eg [here](https://github.com/MetabolicAtlas/ecModels/blob/feat/updated-models/ecHumanGEM/scripts/manualModifications.m)). Would it make sense to remove them from there so they don't interrupt the Matlab log?

This should be addressed by solving issue #72

File: eciSM966/scripts/manualModifications.m [hunk 1]
```diff
@@ -9,6 +9,8 @@
 modifications{1} = [];
 modifications{2} = [];
 
+fprintf('Improving model with curated data...')
```
Comment (mihai-sysbio) [inline eciSM966/scripts/manualModifications.m:12]: Won't this print be added to the Matlab log so nicely cleaned up in the recent GECKO PR?

Comment (IVANDOMENZAIN) [reply eciSM966/scripts/manualModifications.m:12]: @mihai-sysbio, this is a GECKO function that is substituted for each species-specific one available in this repository, therefore the PR in GECKO would just have an impact on `manuaModifications` for yeastGEM.

Comment (mihai-sysbio) [reply eciSM966/scripts/manualModifications.m:12]: I see now.. but this means these printouts should be added to all other `manualModifications.m` for the rest of the models, right?

Comment (IVANDOMENZAIN) [reply eciSM966/scripts/manualModifications.m:12]: Exactly! this is mentioned in issue #72, will take care of it with a new PR in some minutes.

Comment (mihai-sysbio) [reply eciSM966/scripts/manualModifications.m:12]: Brilliant!

**Pull request merged**